ParallelRoadTool/ParallelRoadTool.cs

```diff
@@ -271,7 +271,10 @@ private void OnLocaleChanged()
             XmlSerializer serializer = new XmlSerializer(typeof(NameList));
 
             var assembly = Assembly.GetExecutingAssembly();
-            var resourceName = $"ParallelRoadTool.Localization.{LocaleManager.cultureInfo.TwoLetterISOLanguageName}.xml";
+            // BUG: on Mac and Linux LocalManager.cultureInfo always returns 'en' regardless of game language
+            //var resourceName = $"ParallelRoadTool.Localization.{LocaleManager.cultureInfo.TwoLetterISOLanguageName}.xml";
+            var lang = LocaleManager.instance.language;
```

Comment:
Does this work on Windows too?
If so, you can inline it and have `var resourceName = $"ParallelRoadTool.Localization.{LocaleManager.instance.language}.xml"; ` to minimize changes.